usefulUtilities/sitV2/appResources/appData/sitClientV2.js

```diff
@@ -282,22 +296,29 @@
 
     // Standup functionality
     var WAIT_FOR_USER_TO_STAND_MS = 525;
-    function standUp() {
+    function standUp(callback) {
         if (DEBUG) {
             console.log("standup from ", _this.entityID);
         }
 
         // get the entityID, previous position and orientation 
         var sitCurrentSettings = Settings.getValue(SETTING_KEY_AVATAR_SITTING);
-        var settingsEntityID = sitCurrentSettings[0];
+        var settingsEntityID = sitCurrentSettings;
 
+        MyAvatar.clearPinOnJoint(MyAvatar.getJointIndex("Hips"));
+        changedSeats = false;
+        if (isConnected) {
+            Controller.actionEvent.disconnect(onActionEvent);
+            isConnected = false;
+        }
         // STANDING FROM THIS CHAIR
         // Make avatar stand up (if changed seat do not do this)
         if (settingsEntityID === _this.entityID) { // POSSIBLE RACE CONDITION WITH SETTINGS BEING CHANGED BY NEW SEAT
```

Comment:
what happens if you remove this conditional entirely? now that you've (theoretically) fixed the race condition, shouldn't you not need this anymore?